twitter4j-core/src/main/java/twitter4j/TwitterImpl.java

```diff
@@ -77,6 +77,9 @@
             if (conf.isIncludeExtAltTextEnabled()) {
                 params.add(new HttpParameter("include_ext_alt_text", "true"));
             }
+            if (conf.isTweetModeExtended()) {
+                params.add(new HttpParameter("tweet_mode", "extended"));
```

Comment:
I think an enum would be useful here to illustrate the allowed values of tweet_mode ("compat" and "extended")